### ISSUE TITLE
":all" pseudo-param

### ISSUE THREAD

**Issue by jonleighton** [cjbottaro/param_protected#2]
It would be good to be able to specify:

  param_accessible :all

If there is a param_accessible declaration in a superclass of the controller, then this would allow the params to be open right up again if desired.

**Comment by cjbottaro** [comment]
Good idea, will look into putting that in.  Won't be until next week though, going on vacation tomorrow.
